apps/pie-docs/src/test/test.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ogImage": "landing_support.png"
```

Comment:
are images intended to use underscores, or should we use hyphens?

Reply:
It is what we have been using across the site. I have removed it for now though as it's not necessary for this PR